openshift/ocp_on_openstack.adoc

```diff
@@ -89,6 +89,83 @@ Push the `pass` changes to the git repository.
 git push
 ----
 
+
+== Post installation steps
+
+To be able to SSH to any of the OCP cluster machines a _jump server_ is required.
+
+=== Install jump server
+
+[source,bash]
+----
+ansible-playbook ansible/playbook/openstack/openstack_vm_create_passwordstore.yml \
+  -e '{"openstack": {"vm": {"network": "ocp-sscpc-openshift","image": "Fedora-Cloud-Base-29", "flavor": "m1.small"}}}' \
+  -e vm_name=ocp-jump-server
+----
+
+Create a floating ip from the external network that your router is connected to, in our case `provider_net_cci_13`.
```

Comment:
This point should be clarified as I'm lost between the role of "ocp-sscpc-openshift" and "provider_net_cci_13" !

Reply:
Improving the docs.